airbyte-integrations/connectors/source-opsgenie/source_opsgenie/schemas/integrations.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": ["null", "string"]
+    },
+    "name": {
+      "type": ["null", "string"]
+    },
+    "enabled": {
+      "type": ["null", "boolean"]
+    },
+    "type": {
+      "type": ["null", "string"]
+    },
+    "teamId": {
```

Comment:
Team name is also supported in schema https://docs.opsgenie.com/docs/integration-api